development.ini

```diff
@@ -18,7 +18,7 @@ sqlalchemy.url = sqlite:///%(here)s/back.sqlite
 
 retry.attempts = 3
 
-available_locales = en sl
+available_locales = en si
```

Comment:
While SI is the ISO alpha-2 code for slovenia, SL is the correct language. SL should be used in this context, otherwise browser language detection won't work.

Maybe we should use a library that can convert between the two?
Or maybe just add another list of alpha-2 codes corresponding to the language codes...

Reply:
Also apply the same change to production.ini

Reply:
For now I do it manually, if 'sl' then 'si' hahaha, will be ok for now, it is only for flags really.

Reply:
Deal!

Reply:
Done